neuraxle/hyperparams/distributions.py

```diff
@@ -803,7 +802,7 @@ def rvs(self) -> int:
 
         :return: an integer.
         """
-        return random.randint(self.min_included, self.max_included)
+        return np.random.randint(self.min_included, self.max_included+1)
```

Comment:
Spacing autoformat to respect pep8. Suggested change: 
```suggestion
        return np.random.randint(self.min_included, self.max_included + 1)
```

Reply:
I signed it :)